Source/Core/DolphinQt2/NetPlay/MD5Dialog.cpp

```diff
@@ -110,18 +109,20 @@ void MD5Dialog::SetResult(int pid, const std::string& result)
   m_status_labels[pid]->setText(
       tr("%1[%2]: %3").arg(player_name, QString::number(pid), QString::fromStdString(result)));
 
-  if (m_last_result.empty())
-  {
-    m_check_label->setText(tr("The hashes match!"));
-    return;
-  }
+  m_results.push_back(result);
 
-  if (m_last_result != result)
+  if (m_results.size() >= Settings::Instance().GetNetPlayClient()->GetPlayers().size())
   {
-    m_check_label->setText(tr("The hashes do not match!"));
+    if (std::adjacent_find(m_results.begin(), m_results.end(), std::not_equal_to<>()) ==
```

Comment:
`<algorithm>` needs to be included.

Reply:
Actually this function is part of `<functional>`.

Reply:
`std::not_equal_to` is part of functional (which I missed, good catch), `std::adjacent_find` is within `<algorithm>` however.

Reply:
Ah, yeah.